server/src/test/java/com/qubole/quark/serverclient/client/EndToEndTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2015. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.qubole.quark.serverclient.client;
+
+import com.qubole.quark.serverclient.server.Main;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.sql.*;
+import java.util.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+/**
+ * Created by adeshr on 2/24/16.
+ */
+public class EndToEndTest {
+
+  public static final String h2Url = "jdbc:h2:mem:TpcdsTest;DB_CLOSE_DELAY=-1";
+  public static final String cubeUrl = "jdbc:h2:mem:TpcdsCubes;DB_CLOSE_DELAY=-1";
+  public static final String viewUrl = "jdbc:h2:mem:TpcdsViews;DB_CLOSE_DELAY=-1";
+
+  @BeforeClass
+  public static void setUp() throws SQLException, IOException, URISyntaxException,
+      ClassNotFoundException {
+    setupTables(h2Url, "tpcds.sql");
+    setupTables(cubeUrl, "tpcds_cubes.sql");
+    setupTables(viewUrl, "tpcds_views.sql");
+  }
+
+  public static void setupTables(String dbUrl, String filename)
+      throws ClassNotFoundException, SQLException, IOException, URISyntaxException {
+
+    new Thread(new Main()).start();
```

Comment:
Yes, I have to change it to H2 Db, was waiting for how/which arguments to pass to server.